examples/deserialize-transaction/src/example.ts

```diff
@@ -315,6 +315,9 @@ const decompiledTransactionMessage = await decompileTransactionMessageFetchingLo
 log.info(`[step 3] The transaction fee payer is ${decompiledTransactionMessage.feePayer.address}`);
 
 // And the lifetime constraint:
+if (!('lifetimeConstraint' in decompiledTransactionMessage)) {
+    throw new Error('We expect a lifetime');
+}
```

Comment:
This showcases how this PR affects the developer experience. Here we know there will always be a lifetime constraint since we're fetching a transaction that landed so we need to assert the correct type.